### PR TITLE
Implemented /roll command

### LINKED ISSUE #1195: Feature Request: Random roll command
I think it would be good if it were possible to roll an N-sided die, visible to both players in the chat log. I don't think it deserves a button/prompt in the UI. But a `/roll` command would be nice.

> /roll 3
> _bmatsuo tosses a die [1-3] and rolls a 1_

I think there are some spots where this could provide minor utility.
- Randomizing psi games (or giving the opponent the appearance of randomization).
- Sometimes you don't know what to do.

### PR DESCRIPTION
**Pull request by x3N1GM4x**
User can type "/roll n" to roll an n-sided die and have the result displayed in the chat

Fixes #1195.

### CODE REVIEW

**Comment by danhut** [comment]
Nice work. Worth updating the netrunner-help.cljs too so users know about it. 

Also add Fix #num in the comment so GitHub will auto close the linked issue.

File: src/clj/game/core/io.clj [hunk 1]
```diff
@@ -170,6 +170,9 @@
                                             (rez state side c {:ignore-cost :all-costs :force true}))))}}}
     {:title "/rez-all command"} nil))
 
+(defn command-roll [state side value]
+  (system-msg state side (str "rolls a " value " sided die and rolls a " (+ (rand-int (- value 1)) 1))))
```
Comment (nealterrell) [inline src/clj/game/core/io.clj]: Clojure-fu: instead of `(+ ... 1)`, use `(inc ...)`. :) Likewise, `(dec ...)` instead of `(- ... 1)`.

Comment (x3N1GM4x) [reply src/clj/game/core/io.clj]: Plus, I now realise my logic was wrong anyway! Consider my wrists slapped for not testing thoroughly enough! 😄

**Comment by mtgred** [comment]
👍

**Pull request merged**